force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -158,56 +165,56 @@ export default class rd2EntryForm extends LightningElement {
     };
 
     rd2Service = new Rd2Service();
+    @track rd2State = this.rd2Service.init();
 
     @track error = {};
 
     /***
-    * @description Get the next donation date for this recurring donation
-    */
-    @api 
+     * @description Get the next donation date for this recurring donation
+     */
+    @api
     get nextDonationDate() {
         const localDate = new Date(this._nextDonationDate);
         return new Date(localDate.getUTCFullYear(), localDate.getUTCMonth(), localDate.getUTCDate());
     }
 
     /***
-    * @description Get the Credit Card expiration date, formated as MM/YYYY
-    */
-    @api 
-    get cardExpDate(){
-        const cardExpMonth = getFieldValue(this.record, FIELD_CARD_EXPIRY_MONTH);
-        const cardExpYear = getFieldValue(this.record, FIELD_CARD_EXPIRY_YEAR);
-        return (cardExpMonth && cardExpYear) ? cardExpMonth + '/' + cardExpYear : '';
+     * @description Get the Credit Card expiration date, formated as MM/YYYY
+     */
+    get cardExpDate() {
+        const cardExpMonth = this.rd2State.cardExpirationMonth;
+        const cardExpYear = this.rd2State.cardExpirationYear;
+        return (cardExpMonth && cardExpYear) ? cardExpMonth + "/" + cardExpYear : "";
     }
 
     /***
-    * @description Get the Label for the Card Last 4 field
-    */
+     * @description Get the Label for the Card Last 4 field
+     */
     get cardLastFourLabel() {
-        return (this.fields.cardLastFour) ? this.fields.cardLastFour.label : '';
+        return (this.fields.cardLastFour) ? this.fields.cardLastFour.label : "";
     }
 
     /***
      * @description Get the Label for the Card Last 4 field
      */
     get achLastFourLabel() {
-        return (this.fields.achLastFour) ? this.fields.achLastFour.label : '';
+        return (this.fields.achLastFour) ? this.fields.achLastFour.label : "";
     }
 
     /***
-    * @description Dynamically render the new/edit form via CSS to show/hide based on the status of
-    * callouts to retrieve RD settings and other required data.
-    */
+     * @description Dynamically render the new/edit form via CSS to show/hide based on the status of
+     * callouts to retrieve RD settings and other required data.
+     */
     get cssEditForm() {
         return (!this.isLoading && this.isSettingReady && this.isRecordReady)
-            ? ''
-            : 'slds-hide';
+            ? ""
+            : "slds-hide";
     }
 
     /***
-    * @description Get settings required to enable or disable fields and populate their values
-    */
-    connectedCallback() {
+     * @description Get settings required to enable or disable fields and populate their values
+     */
+    async connectedCallback() {
         getRecurringSettings({ parentId: this.parentId })
```

Comment:
Since `getInitialView` is implemented, I think `getRecurringSettings` can be removed and reference the new function.

Reply:
Eventually we can decommission it, but at the moment `getRecurringSettings` still provides data for a few fields on rd2EntryForm. `customFields`, `isChangeLogEnabled`, `periodToYearlyFrequencyMap` to name a few. I can probably have it fully pulled out of here w/ the next WI that targets the Schedule Component.